pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -58,17 +58,22 @@ var (
 	noScheduleTaints      = []v1.Taint{{Key: "dedicated", Value: "user1", Effect: "NoSchedule"}}
 )
 
+func nowPointer() *metav1.Time {
```

Comment:
Why not add this into `metav1`, named `NowP() *Time`? So other developer can use it.